picturae_importer.py

```diff
@@ -307,7 +307,6 @@ def taxa_author_tnrs(self, taxon_name, barcode):
         self.parent_author = taxon_list[0]
 
 
-
     def create_agent_list(self, row):
```

Comment:
changed logic for adding agent names to agent dict to strip leading and trailing spaces as well as make sure that None and nan are not turned into strings and missed by isna()